cmd/test.go

```diff
@@ -252,6 +252,11 @@ func opaTest(args []string) int {
 			}
 		}
 	} else {
+		//for test coverage with --bundle, retrieve modules from bundles
+		if modules == nil {
+			modules = map[string]*ast.Module{}
+			modules = tester.AggregateBundleModules(bundles, modules)
```

Comment:
With the change described below, we'd do this here:
```suggestion
			modules = tester.AggregateBundleModules(bundles)
```

Reply:
makes sense ! will change it